kuma/static/js/components/modal.js

```diff
@@ -0,0 +1,88 @@
+(function() {
+    'use strict';
+
+    if (!window.mdn) {
+        window.mdn = {};
+    }
+
+    /**
+     * A modal dialog implemetation based on:
+     * https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal
+     */
+    var modalDialog = {
+        /**
+         * Returns `false` if either of the following conditions are true else, returns `true`:
+         * 1. Multi auth is not turned on
+         * 2. The modal we expect is not present
+         * 3. We are on a mobile phone
+         * @param {Object} modal - The modal as an `HTMLElement`
+         * @returns {Boolean}
+         */
+        showModal: function (modal) {
```

Comment:
This is poorly named. the `verbNoun` makes me think it's going to *do something* with the noun.

Reply:
You are 100% correct. Perhaps `shouldShowModal`?